README.md

```diff
@@ -1,63 +0,0 @@
-# Math Magicians
-
-
```

Comment:
- [ ] You initially had README on this branch. Kindly add your README to this branch as well - to keep it that way.